pages/blog/_posts/vercel-cloudflare-wait-until.mdx

```diff
@@ -0,0 +1,126 @@
+---
+heading: When to use Vercel and Cloudflare's waitUntil utility
```

Comment:
I feel like this title might be a bit authoritative. Maybe: "What is waitUntil (Vercel, Cloudflare) and when do I need it?"